src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -519,6 +519,7 @@ void OpenImageIOReader::hashFileName( const Gaffer::Context *context, IECore::Mu
 
 void OpenImageIOReader::hashFormat( const GafferImage::ImagePlug *output, const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
+	ImagePlug::GlobalScope c( context );
```

Comment:
This seems back-to-front. It should be the responsibility of the caller to use an appropriate context, rather than to override the context in here. Otherwise well behaved callers are paying the price of context management twice (or even worse, doing context management where none is necessary at all).